klifs_utils/local/coordinates/complex.py

```diff
@@ -0,0 +1,44 @@
+"""
+klifs_utils
+Utility functions to work with KLIFS data (local)
+
+Complex coordinates.
+"""
+
+from bravado.client import SwaggerClient
+
+from klifs_utils.util import _mol2_path, _mol2_file_to_dataframe
+
+KLIFS_API_DEFINITIONS = "http://klifs.vu-compmedchem.nl/swagger/swagger.json"
+KLIFS_CLIENT = SwaggerClient.from_url(KLIFS_API_DEFINITIONS, config={'validate_responses': False})
```

Comment:
I think you are defining these in all modules, but having a central instance is enough. You can define it in a single file and then import that instance around!

Reply:
Jaime, this is great! I have been wondering about this.

We have now a central file for the KLIFS client :)
`klifs_utils/klifs_client.py`